pkg/options/config.go

```diff
@@ -0,0 +1,85 @@
+package options
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/rs/zerolog/log"
+
+	"github.com/authzed/connector-postgres/pkg/config"
+	"github.com/authzed/connector-postgres/pkg/pgschema"
+	"github.com/authzed/connector-postgres/pkg/streams"
+	"github.com/authzed/connector-postgres/pkg/util"
+)
+
+type ConfigPrinter func(c *config.Config) error
```

Comment:
Any reason this can't just be an io.Writer that's implemented by you using json.Encoder with .SetIndent on it?

Reply:
the printer's interface is `Print(*Config)` and writer is `Write([]byte)`

Reply:
discussed this in dm, not anything to really care about